### PR TITLE
Add GitHub Actions script for releases on 3.2

### PR DESCRIPTION
**Pull request by krashish8**
Changes proposed in this pull request:
- Ports #1857 to 3.2
- Fixed NEWS and release_notes (changed `#1825` to `#1850`)

@pgRouting/admins

### CODE REVIEW

**Comment by krashish8** [comment]
The NEWS was not in proper markdown format, (e.g. bold should be `** **` not `* *`, hyperlinks should be `[ ]( )`). Therefore I modified the Perl script and removed the newline character in `release_script`, so that it is in proper markdown.

**Comment by cvvergara** [comment]
Actually with the Action you created to have automatically posted changes on the tag:
https://github.com/pgRouting/pgrouting/blob/develop/doc/src/release_notes.rst#pgrouting-320-release-notes

TODO:
Create an issue about increasing the version number for sphinx
Relate this PR to that new issue

File: doc/src/release_notes.rst [hunk 1]
```diff
@@ -20,8 +20,7 @@ To see the full list of changes check the list of `Git commits <https://github.c
 pgRouting 3.2.0 Release Notes
 -------------------------------------------------------------------------------
 
-To see all issues & pull requests closed by this release see the `Git closed milestone for 3.2.0
-<https://github.com/pgRouting/pgrouting/issues?utf8=%E2%9C%93&q=milestone%3A%22Release%203.2.0%22>`_ on Github.
+To see all issues & pull requests closed by this release see the `Git closed milestone for 3.2.0 <https://github.com/pgRouting/pgrouting/issues?utf8=%E2%9C%93&q=milestone%3A%22Release%203.2.0%22>`_ on Github.
```
Comment (cvvergara) [inline doc/src/release_notes.rst:23]: Need to add the issue about the version change on sphinx

**Review by cvvergara — CHANGES_REQUESTED** [review]

File: doc/src/release_notes.rst (hunk 1 above)
Comment (krashish8) [reply doc/src/release_notes.rst:23]: Ok, adding it

**Review by cvvergara — APPROVED** [review]

**Pull request merged**